keyboards/doio/kb16/config.h

```diff
@@ -17,5 +17,13 @@
 
 #pragma once
 
+#define NO_ACTION_ONESHOT
+
```

Comment:
Not sure about this change, given rev2 has sufficient flash. 

rev1 previously had `SPACE_CADET_ENABLE = no` which has not been maintained?